cpp/modmesh/buffer/pymod/wrap_SimpleArrayPlex.cpp

```diff
@@ -196,6 +196,10 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArrayPlex : public WrapBase<Wr
 
     friend root_base_type;
 
+#define DECL_MM_EXECUTE_TYPED_ARRAY_METHOD(typed_array_method)          \
```

Comment:
this macro can saves more lines.

Reply:
It's not easy to tell if you are overusing the macros.  We'll see.  Disassemble them back into repeating characters is not difficult, anyway.